### DISCUSSION TITLE
Define Goals And Scope of the Solid Practitioners Group

### DISCUSSION THREAD

**Discussion by jeff-zucker** [solid-contrib/practitioners#1] [Ideas]
I have created a [draft README](https://github.com/solid-contrib/practitioners) and would value your thoughts, especially on the question of our target audience. Please use this discussion area to share what you think and/or submit a PR to change the wording.

**Comment by vid** [comment]
I get a 404 for the draft README.

**Reply by jeff-zucker** [reply]
Sorry, fixed.

**Comment by NoelDeMartin** [comment]
I love the "getting things done with what exists" part :D.

At this point, I think it's ok to have a broader audience as long as it's following that spirit. So it could be apps or servers, comercial or non-comercial, but working with what we have today in the specs and server implementations.

As long as what you're doing can be used today by anyone who's got a Solid POD, I think it would fit the scope.

**Comment by timea-solid** [comment]
I agree with Noel. We can start with meetings and see who is interested and what discussions arise. If there are too many interests and enough participants we can split the topics off to be more focused. 
I'd say: let's see how the first meetings go and listen/collect needs of our current existing practitioners.

Personally, I would love to work on social impact use cases which both focus on developing business models as much as developing social impact outcomes/output.
